.github/workflows/website-checker.yml

```diff
@@ -21,8 +21,10 @@ on:
 jobs:
   # checks that a 'type/docs-cherrypick' label is attached to PRs with website/ changes
   website-check:
-    # If there's a `type/docs-cherrypick` label we ignore this check
-    if: "!contains(github.event.pull_request.labels.*.name, 'type/docs-cherrypick')"
+    # If there's a `type/docs-cherrypick` label or an explicit `pr/no-docs-cherrypick` label, we ignore this check 
+    if: >-
```

Comment:
In case anyone is wondering, I decided to use a '[Folded Block Scalar](https://yaml-multiline.info/)' here so that the block's newlines would be replaced with spaces rather than having to insert `\` as I'm not sure how GitHub expressions deal with those operators.

Reply:
Yaml is awesome! I don't know why people hate on it